evap/student/templates/student_vote_questionnaire_group.html

```diff
@@ -59,8 +59,8 @@ <h4 class="{% if not forloop.first %} mt-3 {% elif contributor %} me-auto {% end
                             <div class="col-answer col-lg-8 col-xl-7 my-auto d-flex">
                                 <div class="vote-inputs tab-row {{ field.field.widget.attrs.choices.css_class }} btn-group" data-bs-toggle="buttons">
                                 {% for choice, color in field|zip:field.field.widget.attrs.choices.colors %}
-                                    <input id="{{ choice.id_for_label }}" class="tab-selectable num-selectable btn-check" name="{{ choice.data.name }}" type="radio" value="{{ choice.data.value }}" autocomplete="off"{% if field.value == choice.data.value %} checked{% endif %}{% if preview %} disabled{% endif %}/>
-                                    <label for="{{ choice.id_for_label }}" class="btn btn-sm btn-light vote-btn vote-btn-{{ color }} d-flex{% if field.value == choice.data.value %} active{% endif %}{% if field.errors %} choice-error{% endif %}">
+                                    <input id="{{ choice.id_for_label }}" class="tab-selectable num-selectable btn-check" name="{{ choice.data.name }}" type="radio" value="{{ choice.data.value }}" autocomplete="off"{% if field.value == choice.data.value|stringformat:"d" %} checked="checked"{% endif %}{% if preview %} disabled{% endif %}/>
```

Comment:
can we add / modify a test to ensure these are properly filled when reloading the page after submitting form values that are not accepted by the server?